app/assets/stylesheets/pageflow/editor/base.scss

```diff
@@ -15,6 +15,7 @@
 @import "./indicators";
 @import "./disabled_atmo_indicator";
 @import "./unsupported_browser_hint";
+@import "./change_theme";
```

Comment:
The ReferenceInputView's button also is styled by this. Should it have the editor class also?